Improve parsing for scapy.contrib.postgres

jrolli opened the pull request:
This fixes two major usability issues with the existing code:
1) Parsing of Bind messages
2) Proper reassembly of multi-packet message groups

Additionally, it makes minor changes to some of the existing code
that was causing bad parsing on real server traffic mostly focused
on length/count calculations.

Known bugs:

- This may error out on asynchronous server-generated messages
    (e.g. NoticeResponse & NotificationResponse) because of the
    simplified logic in `tcp_reassemble`.  I did not have any examples
    of this happening in real traces, so did not address at this time.

- Does not handle FunctionCall messages.  This was not implemented
    in the original code (commented out as a "TODO") but now it is
    replaced with a "_Todo" packet that allows parsing to continue for
    other messages in the stream (message packets follow a consistent
    TLV format).

p-l- reviewed (changes requested):
Some tiny changes are needed.

Also, could you some tests to cover your changes?

Thanks!

jrolli commented:
@p-l-, thanks for the feedback!  I made the two code changes and added a test that checks the class I added (Bind) as well as the one I fixed (Parse).  Let me know if there are any other changes you would like to see.

p-l- commented on test/contrib/postgres.uts:
```diff
@@ -238,3 +238,70 @@ assert copydata_out.contents[6].data == b'6\tPip\t66\t6\n'
+= Check sync decoding
+
+sync_msg = request.contents[4]
+assert sync_msg.len == 4
```
Would you mind adding a newline at the end of this file?

p-l- reviewed (changes requested):
Sorry to be so picky, just another (very) tiny change please!
